astroid/interpreter/objectmodel.py

```diff
@@ -100,20 +121,16 @@ def __get__(self, instance, cls=None):
     def __contains__(self, name):
         return name in self.attributes()
 
-    @lru_cache(maxsize=None)
-    def attributes(self):
+    def attributes(self) -> List[str]:
```

Comment:
The expensive part of this function is (I assume) the iteration over the `dir`. The other two function seem fairly cost efficient? Perhaps just remove the caching here altogether?